gokart/build.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import os
+import sys
+from configparser import ConfigParser
+from logging import getLogger
+from typing import Any, Optional
+
+import luigi
+
+from gokart.task import TaskOnKart
+
+
+class HideLogger:
+    def __init__(self, verbose: bool):
+        self.verbose = verbose
+        self.logger = getLogger(__name__)
+        self.default_level = self.logger.level
+
+    def __enter__(self):
+        if not self.verbose:
+            logging.disable(sys.maxsize)
+            self.logger.setLevel(logging.CRITICAL)
+        return self
+
+    def __exit__(self, exception_type, exception_value, traceback):
+        logging.disable(self.default_level)
+        self.logger.setLevel(self.default_level)
+
+
+def _get_output(task: TaskOnKart) -> Any:
+    output = task.output()
+    if type(output) == list:
+        return [x.load() for x in output]
+    return output.load()
+
+
+def _read_environ():
+    config = luigi.configuration.get_config()
+    for key, value in os.environ.items():
+        super(ConfigParser, config).set(section=None, option=key, value=value.replace('%', '%%'))
+
+
+def add_config(file_path: str):
+    _, ext = os.path.splitext(file_path)
+    luigi.configuration.core.PARSER = ext
+    assert luigi.configuration.add_config_path(file_path)
+
+
+def build(task: TaskOnKart, verbose: bool = False, return_value: bool = True, reset_register: bool = True) -> Optional[Any]:
+    """
+    Run gokart task for local interpreter.
+    """
+    if reset_register:
+        luigi.task_register.Register._reg = [x for x in luigi.task_register.Register._reg
+                                             if x.__module__.split('.')[0] in {'gokart', 'luigi'}]  # avoid TaskClassAmbigiousException
+    _read_environ()
```

Comment:
[ASK]
Why don't we use gokart.run._read_environ() ?

Reply:
[ASK]
Is it better to add _check_config as like gokart.run() ?
https://github.com/m3dev/gokart/blob/7056c10cf4577d7008b5744c5cf127da5b65c849/gokart/run.py#L110

Reply:
[ASK]
Is there a chance to call gokart.build() from gokart.run() ?
Cause there are many overlapping functions.

Reply:
All right

Reply:
It has an underscore, so I instinctively avoided it. `gokart.run._check_config` is good too.

Reply:
> Is there a chance to call gokart.build() from gokart.run() ?

Not supposed to. These are both `endpoints`.